integration/nwo/standard_networks.go

```diff
@@ -86,6 +75,31 @@ func BasicSolo() *Config {
 	}
 }
 
+// FullSolo is a configuration wtih two organizations and two peers per org.
+func FullSolo() *Config {
```

Comment:
Not a big deal, but I'd prefer more descriptive names rather than BasicSolo() and FullSolo(), e.g. SoloWithTwoOrgsOnePeerEach() and SoloWithTwoOrgsTwoPeersEach(). That way when looking at a test you know exactly what you are dealing with.